### DISCUSSION TITLE
Hello December! Actions & Packages check-in 📰

### DISCUSSION THREAD

**Discussion by Akash1134** [community/community#40607] [Actions]
Hello 👋 Y’all, It’s December! 

A new month, a new beginning, and a new focus. This week, I bring you some thrilling new updates, product releases, and love from all corners of the Actions and Packages community. Let’s dive into this!

## [GitHub Actions Importer](https://github.blog/changelog/2022-11-16-introducing-github-actions-importer/) - CI/CD migration made easy 🆕 

We’ve launched a public preview of GitHub Actions Importer, which helps you forecast, plan, and automate migrations from your current CI/CD tool to GitHub Actions. GitHub Actions Importer gets you up and running faster by planning and executing migrations from your former CI/CD tool to GitHub Actions.

Sign up [here](https://github.com/features/actions-importer/signup) to request access to the public preview. Once added, you will receive an email with instructions to get started.

To learn more, see [Automating Migrations with GitHub Actions Importer](https://docs.github.com/actions/migrating-to-github-actions/automating-migration-with-github-actions-importer) and the [announcement](https://github.blog/2022-11-10-introducing-github-actions-importer/) on the GitHub blog.

## [Revolutionize Internet of Things (IoT) software development](https://github.blog/2022-11-02-github-partners-with-arm-to-revolutionize-internet-of-things-software-development-with-github-actions/)

GitHub has partnered with Arm to revolutionize IoT software development by making the Arm Development tools (Arm cross-compiler and [Arm Virtual Hardware](http://github.com/Arm-software/AVH-GetStarted)) natively available inside GitHub Actions cloud hosted runners to create an efficient CI workflow. This [in-depth article](https://resources.github.com/devops/actions/arm-virtual-hardware/) has all of the steps you need to run Arm Virtual Hardware inside GitHub Actions.
 
### Require approval for running workflows from private forks. ✔️ 

You can now require approval from a user with write permissions to the repository before a workflow run can be triggered from a private fork. To know more about this ship 🚢, see: [Enabling workflows for forks of private repositories](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/enabling-features-for-your-repository/managing-github-actions-settings-for-a-repository#enabling-workflows-for-forks-of-private-repositories)

### [Ubuntu-latest workflows will use Ubuntu-22.04](https://github.blog/changelog/2022-11-09-github-actions-ubuntu-latest-workflows-will-use-ubuntu-22-04/) 💻 

Ubuntu 22.04 became [generally available on GitHub-hosted runners](https://github.blog/changelog/2022-08-09-github-actions-ubuntu-22-04-is-now-generally-available-on-github-hosted-runners/) in August 2022. Now Ubuntu-22.04 is ready to be the default version for the ubuntu-latest label in GitHub Actions workflows.

### [Webhook enhancements for environment protection rules](https://github.blog/changelog/2022-11-22-webhook-enhancements-for-environment-protection-rules/) ♻️ 

The workflow_job webhook will now support a new waiting state whenever a job is waiting on [an environment protection rule](https://docs.github.com/en/actions/deployment/targeting-different-environments/using-environments-for-deployment#environment-protection-rules), aligning with the waiting state of the corresponding check run.

### [Restrict workflows to specific runners using runner group names](https://github.blog/changelog/2022-11-01-github-actions-restrict-workflows-to-specific-runners-using-runner-group-names/) 🔒 

You can now deterministically restrict workflows to run on a specific set of runners using the names of their runner groups in the runs-on key of their workflow YAML. This prevents the unintended case where your job runs on a runner outside your intended group because the unintended runner shared the same labels as the runners in your intended runner group.

##  From the Community ❤️ 

Let’s give a round of applause 👏 to these community members for their contributions.

- A brilliant self-solve 💡 by [hiddewie](https://github.com/hiddewie) Thanks for coming back with the [solution ✅ ](https://github.com/orgs/community/discussions/38570) that worked for you and sharing the update. 
- Another [self-solve](https://github.com/orgs/community/discussions/40423) by [riccardoporreca](https://github.com/riccardoporreca) where he troubleshot a permission issue with ease.
- [Laughedelic](https://github.com/laughedelic) showcased the power 💪 of Docs, and helped [ldeluigi](https://github.com/ldeluigi) by answering his [question](https://github.com/orgs/community/discussions/39097).
- Shoutout 🎉 to [david-engelmann](https://github.com/david-engelmann) for doing a great job solving this [question](https://github.com/orgs/community/discussions/38510) about Docker containers.

Want to flex some solving muscled? Help us with these unanswered threads from the month of November 🤝 
  - [Delete branch cache on branch deletion](https://github.com/orgs/community/discussions/38807)
  - [Making all CI checks required](https://github.com/orgs/community/discussions/38994)
  - [How to create multiple groups with workflows inside it](https://github.com/orgs/community/discussions/38909)
  - [Authorized user cannot push in GitHub Action](https://github.com/orgs/community/discussions/39189)

That's all for now folks. Got feedback or feature requests for anything Actions and Packages? ❓Post them on [community](https://github.com/github-community/community/discussions/categories/actions-and-packages) and share enhancements you’d like to see on these products. Happy December! Till next time! 🚀

**Comment by Trnvnphuong20** [comment]
Phuong
